### ISSUE TITLE
bookify crashed with ImportError in <module>()

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3338]
|<img alt="doko's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| doko reported|
|-|-|
|Trac ID|trac#3338|
|Type|defect|
|Created|2008-07-08 13:38:56Z|
|Branch|https://github.com/twisted/twisted/tree/remove-bookify-3338|

forwarded from https://bugs.launchpad.net/bugs/212830

Title: bookify crashed with `ImportError` in <module>()
```
Traceback:
 Traceback (most recent call last):
   File "/usr/bin/bookify", line 20, in <module>
     from twisted.lore.scripts.bookify import run
 ImportError: No module named bookify

can't find any bookify module
```

<details><summary>Searchable metadata</summary>

```
trac-id__3338 3338
type__defect defect
reporter__doko doko
priority__normal normal
milestone__ 
branch__branches_remove_bookify_3338 branches/remove-bookify-3338
branch_author__ 
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__ 
time__1215524336000000 1215524336000000
changetime__1215700739000000 1215700739000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

bookify was deleted years ago, apparently incompletely.  We should delete the bookify script to complete that removal.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [24242]) Branching to 'remove-bookify-3338'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [24243]) I remove it

refs [#3338](https://github.com/twisted/twisted/issues/3338)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_7"></a>|@therve set owner to @exarkun|
|-|-|

Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set status to `closed`|
|-|-|

(In [24256]) Merge remove-bookify-3338

Author: exarkun
Reviewer: therve
Fixes: [#3338](https://github.com/twisted/twisted/issues/3338)

Remove the vestigal `bookify` command line tool which is part of the Lore
book generation functionality, the rest of the implementation of which was
deleted years ago.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
